desktop/api/src/main/java/org/datacleaner/util/IconUtils.java

```diff
@@ -188,6 +188,10 @@
     public static final String DICTIONARY_TEXTFILE_IMAGEPATH = "images/model/dictionary_textfile.png";
     public static final String DICTIONARY_DATASTORE_IMAGEPATH = "images/model/dictionary_datastore.png";
 
+    public static final String NEWS_CHANNEL = "images/news/news_channel_16.png";
+    public static final String NEWS_CHANNEL_ORANGE = "images/news/news_channel_28.png";
```

Comment:
(I know we do something similar in other places, but we're trying to move away from that)